### PR TITLE
initial cut of profile listener

### PR DESCRIPTION
**Pull request by beav**
This reads messages off of the egress queue, pulls a system profile
for the updated host, and POSTs to the historical system profile
service.

This commit is just a first cut; the listener will be updated in
future commits.

### CODE REVIEW

File: system_profile_archiver/config.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+AWS_SECRET_ACCESS_KEY = os.getenv("CW_AWS_SECRET_ACCESS_KEY", None)
+AWS_REGION_NAME = os.getenv("CW_AWS_REGION_NAME", "us-east-1")
+INVENTORY_URL = os.getenv("INVENTORY_URL", "inventory_url_not_set")
+HISTORICAL_SYS_PROFILE_URL = os.getenv(
+    "HISTORICAL_SYS_PROFILE_URL", "historical_sys_profile_url_not_set"
+)
+LOG_GROUP = os.getenv("LOG_GROUP", "platform-dev")
+PROMETHEUS_PORT = int(os.getenv("PROMETHEUS_PORT", 8000))
+DISABLE_PROMETHEUS = True if os.getenv("DISABLE_PROMETHEUS") == "True" else False
+NAMESPACE = get_namespace()
```
Comment (ehelms) [inline system_profile_archiver/config.py:42]: Should we group all of these at the top with the other constants?

File: system_profile_archiver/logging.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        stream_name=config.NAMESPACE,
+    )
+    cw_handler.setFormatter(LogstashFormatterV1())
+    logger.addHandler(cw_handler)
+
+
+def initialize_logging():
+    kafkalogger = logging.getLogger("kafka")
+    kafkalogger.setLevel("ERROR")
+    if any("KUBERNETES" in k for k in os.environ):
```
Comment (ehelms) [inline system_profile_archiver/logging.py:31]: A blank line above here would help readability

File: app.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                )
+                system = systems[0]["system_profile"]
+                system["system_profile_exists"] = True
+                system["display_name"] = systems[0]["display_name"]
+                profile = {"inventory_id": inventory_uuid, "profile": system}
+                headers = {
+                    "x-rh-identity": identity,
+                    "content-type": "application/json",
+                }
+                result = requests.post(
```
Comment (ehelms) [inline app.py:42]: Could we break this out to its own method or put a blank line above to help separate the logic ? I'm also wondering if that would make testing easier.

**Comment by ehelms** [comment]
Is there any unit testing we can do to ensure basic workflow of the functionality? Or at least apply pylint to the code?

**Comment by beav** [comment]
closing and re-opening; github is being weird about updating this PR

**Pull request closed without merging**